spec/rubocop/cop/layout/line_length_spec.rb

```diff
@@ -37,9 +42,13 @@
   end
 
   it 'registers an offense for long line before __END__ but not after' do
-    inspect_source(['#' * 150,
-                    '__END__',
-                    '#' * 200].join("\n"))
+    maximum_string = '#' * 80
+    expect_offense(<<~RUBY, maximum_string: maximum_string)
+      #{maximum_string}#{'#' * 70}
+      _{maximum_string}#{'^' * 70} Line is too long. [150/80]
+      __END__
+      #{'#' * 200}
+    RUBY
     expect(cop.messages).to eq(['Line is too long. [150/80]'])
```

Comment:
This message test (L52) looks like it can be removed :-)

Reply:
Yeah, thanks! 
Now I see - I missed it in a bunch of places :\

Reply:
Fixed